packages/design-system-tokens/src/themes/cmsgov-component.ts

```diff
@@ -449,11 +449,20 @@ export const components: AnyTokenValues = {
 
   'usa-banner': {
     '__background-color':                         t.color['gray-lightest'],
-    '__color':                                    t.color['black'],
-    '-close__background-color':                   t.color['gray-light'],
-    '-action__color':                             t.color['primary-darker'],
-    '-lock-icon__color':                          t.color['success-light'],
```

Comment:
Changed this from `-lock-icon__color` to `-security-icon__color` - figured it'd be easier to maintain if we decided to change icons in the future. What if the icon stopped being a lock at some point? Might be over-engineering here...